CHANGELOG.txt

```diff
@@ -1,3 +1,25 @@
+=== 0.7.0 (27-08-2017) ===
+
+* Added the feature to allow moderators to send email notifications to students.
+* Fixed activation key bug in email verification.
+* Improved the error display of the stdIO type questions in Yaksh.
+* Add more tests for views.
```

Comment:
I suggest all these messages should be in present tense, with an imperative verb.